DirectProgramming/Fortran/CombinationalLogic/openmp-primes/README.md

```diff
@@ -0,0 +1,98 @@
+# `OpenMP Primes`
+This sample is designed to illustrate how to use 
+the OpenMP* API with the Intel® Fortran Compiler.
+
+This program finds all primes in the first 40,000,000 integers, 
+the number of 4n+1 primes, and the number of 4n-1 primes in the same range. 
+It illustrates two OpenMP* directives to help speed up the code.
+
+  
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | macOS* with Xcode* installed 
+| Software                          | Intel&reg; oneAPI Intel Fortran Compiler (Beta)
+| What you will learn               | How to build and run a Fortran OpenMP application using Intel Fortran compiler
+| Time to complete                  | 10 minutes
+
+## Purpose
+
+This program finds all primes in the first 40,000,000 integers, the number of 4n+1 primes, 
+and the number of 4n-1 primes in the same range. It illustrates two OpenMP* directives 
+to help speed up the code.
+
+First, a dynamic schedule clause is used with the OpenMP* for directive. 
+Because the for loop's workload increases as its index gets bigger, 
```

Comment:
Change:
 Because the for loop's workload
To
Because the DO loop’s workload

Reply:
Barbara caught this.  fixed will push end of day 8/24